v3/util/credential-slave.sh

```diff
@@ -9,7 +9,9 @@ secretslave=$(etcdctl get /frameworkauth/secret)
 
 mkdir $HOMEDIR/mesos-slave
 touch $HOMEDIR/mesos-slave/passwd
-
+sudo cp /root/.dockercfg /home/core/.dockercfg
+sudo chown core:core /home/core/.dockercfg
+chmod 0600 /home/core/mesos-slave/passwd
```

Comment:
I don't think this file exists till line 17, also, what user does this file get run as, root / core? if we are doing a `chmod` it should probably use `sudo` as well if this is not being run as root.